theme/icons.go

```diff
@@ -305,11 +311,36 @@ func WarningIcon() fyne.Resource {
 	return warning
 }
 
-// FileIcon returns a resource containing the standard file icon for the current theme
+// FileIcon returns a resource containing the appropriate file icon for the current theme
```

Comment:
I think "default" is more accurate than "appropriate" as this method call is not dynamic so it can't really tell what is appropriate...